### DISCUSSION TITLE
Link to the Discord?

### DISCUSSION THREAD

**Discussion by ghost** [chakra-ui/chakra-ui#3750] [Q&A] [answered]
Hi,

Was just trying to access the discord via the links provided on chakra-ui site, but they were invalid.

Can someone provide a working link to the discord (if it is still active)?

Thanks!

**Comment by malashevskyi** [comment]
[link](https://discord.com/invite/dQHfcWF)

**Reply by ghost** [reply]
Weird, I am also getting invite invalid for this link as well

**Comment by nikolovlazar** [comment]
Thanks for mentioning this! Did you try to access the Discord link from the navigation bar?

**Reply by ghost** [reply]
Yes - both from the navigation bar, and from a link in the docs, here: https://chakra-ui.com/docs/migration (bottom of the page)

**Reply by nikolovlazar** [reply]
It does open on my side. None of these work? It's the same link that @malashevskyi posted, which actually works. Maybe it's the browser?

**Reply by ghost** [reply]
🤦‍♂️ This was due to an error on my side - I had an unconfirmed account logged in. Sorry for the noise and thank you for your help!

**Comment by ghost** [comment, marked as answer]
User error 🤦  Sorry for the noise
